__tests__/convertColor.test.ts

```diff
@@ -0,0 +1,52 @@
+import { 
+	figmaRGBToWebRGB,
+	webRGBToFigmaRGB,
+	figmaRGBToHex,
+	hexToFigmaRGB
+ } from '../src'
+
+
+const figmaRGB = { r: 0, g: 0.1, b: 1 }
+const figmaRGBA = { r: 0, g: 0.1, b: 1, a: .5 }
+
+const webRGB: [number, number, number] = [0, 26, 255]
+const webRGBA: [number, number, number, number] = [0, 26, 255, .5]
+
+const HEX = "#001aff"
+const HEXa = "#001aff80"
+
+describe('convertColor', () => {
+	test('figmaRGB to webRGB', () => {
+		expect(figmaRGBToWebRGB(figmaRGB)).toEqual(webRGB)
+	})
+	test('figmaRGBA to webRGBA', () => {
+		expect(figmaRGBToWebRGB(figmaRGBA)).toEqual(webRGBA)
+	})
+	test('webRGB to figmaRGB', () => {
+		expect(webRGBToFigmaRGB(webRGB)).toEqual({ r: 0, g: 0.10196078431372549, b: 1 })
+	})
+	test('webRGBA to figmaRGBA', () => {
+		expect(webRGBToFigmaRGB(webRGBA)).toEqual({ r: 0, g: 0.10196078431372549, b: 1, a: 0.5 })
+	})
+	test('figmaRGB to HEX', () => {
+		expect(figmaRGBToHex(figmaRGB)).toBe(HEX)
```

Comment:
This is a good example of test, because you compare result with predefined value `HEX`